code/modules/cargo/gondolapod.dm

```diff
@@ -0,0 +1,69 @@
+/mob/living/simple_animal/pet/gondola/gondolapod
+	name = "gondola"
+	real_name = "gondola"
+	desc = "The silent walker. This one seems to be part of a delivery agency."
+	response_help = "pets"
+	response_disarm = "bops"
+	response_harm = "kicks"
+	faction = list("gondola")
+	turns_per_move = 10
+	icon = 'icons/mob/gondolapod.dmi'
+	icon_state = "gondolapod"
+	icon_living = "gondolapod"
+	pixel_x = -16//2x2 sprite
+	pixel_y = -5
+	layer = TABLE_LAYER//so that deliveries dont appear underneath it
+	loot = list(/obj/effect/decal/cleanable/blood/gibs, /obj/item/stack/sheet/animalhide/gondola = 1)

```

Comment:
This one's missing `/obj/item/reagent_containers/food/snacks/meat/slab/gondola = 1`

Reply:
@81Denton gondola pod is big and strong so it gets 2